client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -655,20 +658,32 @@ export const ApplicationsTable: React.FC = () => {
     : [];
   const applicationDeleteDropdown = applicationWriteAccess
     ? [
-        <DropdownItem
-          key="manage-applications-bulk-delete"
-          isDisabled={
+        <ConditionalTooltip
+          isTooltipEnabled={
             selectedRows.length < 1 ||
-            selectedRows.filter(
+            selectedRows.some(
```

Comment:
I think we want to delete the app even though it is not selected?

Reply:
In the single delete, we shouldn't be dealing with selection state at all I don't think?

Reply:
Oh I see. My fault. Line 466 smay be the cuprit then.

Reply:
@ibolton336, for me the single delete is on line #476 for that file.